indico/htdocs/js/utils/forms.js

```diff
@@ -78,8 +97,37 @@
         }).on('change input', function() {
             var $this = $(this);
             var untouched = $this.serialize() == $this.data('initialData');
+            var $cornerMessage = $('.corner-message');
             $this.find('[data-disabled-until-change]').prop('disabled', untouched);
             $this.closest('form').data('fieldsChanged', !untouched);
+            if ($this.find('[data-save-reminder-message]').length) {
+                if (!isElementInView($this.find('[data-save-reminder-message]'))
+                && !untouched && $cornerMessage.length == 0) {
```

Comment:
Yes, but I saw `==` in other lines of the file and I decided to be consistent. Should I replace them all for `===`/`!==`?